client/package.json

```diff
@@ -14,7 +14,9 @@
     "axios": "^0.19.2",
     "classnames": "^2.2.6",
     "cors": "^2.8.5",
-    "cypress": "^4.2.0",
+    "cypress": "^5.0.0",
```

Comment:
in other repos I think we were able to get working with at least `cypress@5.3.0`? Is it possible to get closer to latest?